pkg/client/cmd/run.go

```diff
@@ -97,10 +98,19 @@ func (*runCmdOptions) validateArgs(cmd *cobra.Command, args []string) error {
 		return errors.New("accepts 1 arg, received " + strconv.Itoa(len(args)))
 	}
 	fileName := args[0]
-	if _, err := os.Stat(fileName); err != nil && os.IsNotExist(err) {
-		return errors.New("file " + fileName + " does not exist")
-	} else if err != nil {
-		return errors.New("error while accessing file " + fileName)
+	if !strings.HasPrefix(fileName, "http://") && !strings.HasPrefix(fileName, "https://") {
+		if _, err := os.Stat(fileName); err != nil && os.IsNotExist(err) {
+			return errors.New("file " + fileName + " does not exist")
+		} else if err != nil {
+			return errors.New("error while accessing file " + fileName)
+		}
+	} else {
+		resp, err := http.Get(fileName)
+		if err != nil {
+			return errors.New("The URL provided is not reachable " + fileName)
```

Comment:
You can use `errors.wrap` here (from the other `errors` package) so you don't lose the error cause